Source/Obsolete/MatcherAttribute.cs

```diff
@@ -157,6 +157,8 @@ namespace Moq
 	/// </code>
 	/// </example>
 	[AttributeUsage(AttributeTargets.Method, Inherited = true)]
+	[EditorBrowsable(EditorBrowsableState.Never)]
+	[Obsolete("This feature has been deprecated in favor of `Match.Create`.")]
```

Comment:
An existing remark in the documentation comment already states that this type has been deprecated, this just formalises it.